### ISSUE TITLE
react v15.0.0 warning

### ISSUE THREAD

**Issue by Jakehp** [kirjs/react-highcharts#127]
```js
npm WARN react-highcharts@8.1.1 requires a peer of react@~0.14 but none was installed.
npm WARN react-highcharts@8.1.1 requires a peer of react-dom@~0.14 but none was installed.
```

**Comment by Jakehp** [comment]
Pending
https://github.com/kirjs/react-highcharts/pull/125
https://github.com/kirjs/react-highcharts/pull/126

**Comment by kirjs** [comment]
Should be fixed in 8.2.0
